.github/workflows/builds.yml

```diff
@@ -206,11 +204,11 @@ jobs:
           && source build/python/setup.sh
           && export LD_LIBRARY_PATH=$PWD/build/thirdparty/OpenDataDetector/factory:$LD_LIBRARY_PATH
           && pip3 install -r Examples/Python/tests/requirements.txt
-          && pytest -rFs
+          && pytest -v
```

Comment:
I think `-rFs` is a little cleaner because it doesn't report everything. Can we do both at least?

Reply:
sure - I think with `-rFs` I was not able to see which unit test failed only which file. that made it hard for me to cross check with my local testing